shell/common/api/remote/remote_object_freer.cc

```diff
@@ -57,10 +59,13 @@ void RemoteObjectFreer::RunDestructor() {
   if (!render_frame)
     return;
 
-  mojom::ElectronBrowserPtr electron_ptr;
+  mojo::PendingRemote<mojom::ElectronBrowser> electron_pending;
   render_frame->GetRemoteInterfaces()->GetInterface(
-      mojo::MakeRequest(&electron_ptr));
-  electron_ptr->DereferenceRemoteJSObject(context_id_, object_id_);
+      electron_pending.InitWithNewPipeAndPassReceiver());
```

Comment:
`PendingRemote` is useful when you want to hold the pipe while transfering it to another sequence or thread. Here it can just be `Remote`

```
mojo::Remote<mojom::ElectronBrowser> electron_browser_remote;
render_frame->GetRemoteInterfaces()->GetInterface(
    electron_browser_remote.BindNewPipeAndPassReceiver());
electron_browser_remote->DereferenceRemoteJSObject(....);
```